applications/EigenSolversApplication/tests/test_eigen_direct_solver.py

```diff
@@ -15,15 +16,23 @@ def _execute_eigen_direct_solver_test(self, class_name, solver_type):
         space = KratosMultiphysics.UblasSparseSpace()
 
         settings = KratosMultiphysics.Parameters('{ "solver_type" : "' + solver_type + '" }')
-        
+
         solver = ConstructSolver(settings)
 
         a = KratosMultiphysics.CompressedMatrix()
-        
-        KratosMultiphysics.ReadMatrixMarketMatrix('../../../kratos/tests/A.mm', a) # symmetric test matrix
-        
+
+        this_file_dir = os.path.dirname(os.path.realpath(__file__))
+        base_dir = os.path.dirname(os.path.dirname(os.path.dirname(this_file_dir)))
+        matrix_file_path = os.path.join(base_dir, "kratos")
+        matrix_file_path = os.path.join(matrix_file_path, "/tests")
```

Comment:
`/` works only on linux afaik
is adding this necessary? From what I see it should also work without (you are already using `os.path.join`)

Reply:
that was a copy/paste error. Thanks for spotting

Reply:
fixed it

Reply:
> `/` works only on linux afaik

Works for Windows too